consensus/qbft/core/backlog.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2017 The go-ethereum Authors
```

Comment:
The initial backend work for both `istanbul` and `qbft` consensus is common, hence instead of duplicating that piece of code, `qbft` uses that as well. There are few changes though around how the `extradata` fields are populated and verified, hence `istanbul` package is impacted